lib/hardware_services/network_drivers/QuadsNativeNetworkDriver.py

```diff
@@ -11,6 +11,9 @@
 import logging
 from subprocess import call
 from subprocess import check_call
+from os import path
+
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
```

Comment:
Yes. I took it from
https://github.com/BUEC528-HIL-QUADS/quads/blob/master/lib/hardware_services/inventory_drivers/HilInventoryDriver.py